img/transaction_flow/api-flow-diagram

```diff
@@ -0,0 +1 @@
Ucu+pCmpVT0TVov2rL59oy7efb0RHWEN15ySZrbGJyevzEuXw6woUUPBMLVZbnDcKl4IoG8XFEnInxMUrDcJzbT4M8cyyYTpcfSTMsKyfGM9HwuyJ8bYzKcar81VhPIEBWiFYKGgKEsiJRrCkFEnRU8qbcpSWOtZQbP7ex4B2a6uuavqlYba01TAHAveqovHB3FTE5aEhJfW8hzCn1j/jQ4wiRJX52Nh+YTqm/l8qWaG4vcJZBfXaU3UnO3fYN6ZGfVcVi0lR32hhfm0Cxkd9qy/qHwj63Uhg+W3BcAUfEFXwfpVvDY8OqjG5IRDkCtU/FhI81o1mQv/zVtl5K75jTa35qotoFwUPMFvRAWYzrz02p15PMRPYoJCLkP0qnQHuF0JCMGFmA6WVl0iqug4Z/gjFcZMI+AEMAlqe5QR/h407c9+xHX2YCatCS0Ih+06YP9aEiQ6dClRPR9C9nlDtjwLVjgTVvnFcqDZUt7yhs22V/XSriMj7pPCG3j3XNYXXHasdoSkBcF9pGJ+xe8o8m7qhTHGFxxNbONuSANOe2sKZqqJ8UMD0ewImd3OHR0zZ69F7stsrQorIs3isxJuNDXl2NzTXUsRrHFRN3a7AGaeg61jmyTESJW0tBjFV45B9K3xZIVI73bRL9TvNWhpSLKgwg3atRHNKuvUYRh2B0U0NR1zOd7XLSz6l2TqKUKCED6R4cGc1tLA5Mr0NFOGgMb59Kh75gjNUDJJ1Wwxk3e4v0u2q/eHt2g7QpXaNWoV2cEpEeg5U2tmss9bxzZ7rnARsTwC/7qGxhYPMpa8MexUCaEYH54j9v1gWZIhx0bkiOEXA8JZ1rQoWlF3OtLAsPq94gcymPH9qNiVC4ErAoLQnKG8ECbgtuVTI6jMqGuI2rrimerlmb2TrIIBZFjHD0CFvFHWLG4XAEfjnGpULHdl6nqD8YAI8SYzBlT3JvpNsGKNxZtXpOHPwNdy+5HfSlA1Thd8pBv4EnJXeE+Zt0K5DQ/G67K3Yxw3wiNywUxGF6dZkeosCD84OLgqG5Lj6wtc7kiR4KhVOwOsfAfOwaEX3ili8z+Ceu2F2ayfowFhelWslmdXxnBzfm0SlX1DOwd10ystvjVu1QrMLoc/7mgSYhlcstZYJXwyyDAW88J5xwOnMvO8OARVvjUyZXhtJZoaUhisjz2zr8+NEsmYqxYchKhX2o4CNv4EGVnmetmUZrbmxhgGf9uqEcbTEIKE5GzKD7ti6rjL3BzlXUWR2O1ehO3cddxjnypRjwWL5Z7oVd1W9pjACA4KuWBPfnTRzKFU/aOC903Bq9zakldVuGu3KTnca7Skm2RjEsirAKAdJLImD8w9QOLhqguWQ2kiLUG535G5Tt2R3U3p+HEvvqdEGuzRiNfITmACUVik9sNMQvE8S4G0nAUVGsNgUJRZIjEFogJR2pEkVjEcDLDXq0KQBN9znPFpkv2qbRfRDGMMcag3hPJNo/ulwDzUXwmTAI8XnH6BYBR6Q74UO9EO7i+/55txyD+V7SphuuiUSlWc0VehrY4njfIZ3w4LUCQy36NGp0Lo3xkkFrRPRl8Zbx6B1Xt+trCPROsuVBUrmY32JnSt5eobZM2D/VmZnKysD3nZuJ78g4sfjcjtVLUof3gUJg5J0nq26rOL7pHLW7sxQd+aPQeU0Q6rVPRqXO41IwyaH1+iJjG2Hd3uUehJkPHTFuOcCpKHviYwKxo6FjGqX7R3IONv+wkjQqKZqlqTso4ChuRUMmVvbhsKhFtaO58aeWqxvP1I4EfT13QQ5EvBJi2SetSfuKTG42Uiwt2HX5qggZp+3hA8orbbTU1ztcdJDbGn6Db+fmA0mCWoU91d8kpYsW4EApQta6lYvYBJCIh5OcQpHsXj2zkiu5bUP2BkojttZ6fj2zzlt6n/EPMLe6DBSGqEEDuaRwcFRue43+FF4rrOd57KMMceT3HPzML0fX7XtHhtjyBIn83X25iByFEF3Q06GN5vrAyW8K/6oSHVtxJDtjhCyPdqyiLro9SvOUcT6HKyznDZD6E8csYrSABPCTo+UNrs0jswR+xNE385rP4LCZ5ypQ/zOqR08ORnXdfoeeuCMc+qBLBu27JuNbc5U8PvJdTu15QNx3dPYKDGFlm84se44+XSuusL0gajlzs0Ihnfg6YBH0JweMaTzopJ2x1F6R6WSjhqMeaQAYupr9k+9o5VTRFgf1RnFVA6LA7WKNLvqAAV9DiNcPvM++KStT80nXTXX/cwthmv3tBjTbm91u4g8z9SjfCltzYDI7StEWsu4TLPzTFJMEsCaKoaQaopAfi3gAsweIov530wWGaUyRPuqF0ue4rej/13NHew6VUmkwdE5eU3KM07yHCYrthwgtrFndNDzRsIh/5Ihcg4PsWz753TNDHti/RQxmVYGXSf49TVOPUZyEvslL1EZescBg37H2I92vqB/KgeAV9sGS0ep3jPIqgxBtqzUaEAg9c2eQDop9Ra9bKeY4oRilDgsWdfY59LrInSVgBQsYAlhb0jWDnGQbcjWZhnZjQCYFsV0toq8bRRqlN5olPdDQudco5+ioO06Y84xlXs6IcXXZr1StrMlWMFCJnK4W+tr0/B5nceIiVFRHgLy/TPzz/MiRHOpO5shopS+t29ey7hom0MdBSP57I7ZwaKsoyKFasw7hVDdLfBbcUblcGIIwrA4hIvKWgRBTil1xiVRnBXOTrHXYMrOi3lVZfF5CQuC3fiDJz9F8Q2iOLkgnsZRYW8yWYemi4vffdLF2cUXSBAdbRZuGfxYyb6+h3mgydwUkpaAcd8Ecubwbq1o5Nj27FRWOXYe2HCmaW3+xEcKyac5GMoBXmNLmAqUj2ixgMWB7bWLy45jU412Cp8v2jtDqwXPDxHxne1cVnEq521fQan2CBxt8cRXQ2G/rUJ+3mNj19ok+z1X7I8VneF2T/GCHFcRea0jxFXoZf3HIbls1H9h07r7Cw==</diagram></mxfile>
```

Comment:
In the docs we started proposing the use of the createAndFund one-call api, and enforce the usage of external IDs for both senders and transactions. We're also having a chat to force-enable auto cancel to any new MTO customer and making it the default. We'll also need to have a look at our current cancel API as it's awful

Reply:
Yeah totally agree just wanted to make sure this diagram got in so we can iterate on this.